bigquery/schema.go

```diff
@@ -176,6 +181,32 @@ func (fs *FieldSchema) toBQ() *bq.TableFieldSchema {
 	return tfs
 }
 
+// RangeElementType describes information about the range type.
+type RangeElementType struct {
+	// The subtype of the RANGE, if the type of this field is RANGE.
+	// Possible values for the field element type of a RANGE include:
+	// DATE, DATETIME, or TIMESTAMP.
+	Type FieldType
+}
+
+func (rt *RangeElementType) toBQ() *bq.TableFieldSchemaRangeElementType {
+	if rt == nil {
+		return nil
+	}
+	return &bq.TableFieldSchemaRangeElementType{
+		Type: string(rt.Type),
+	}
+}
+
+func bqToRangeElementType(bq *bq.TableFieldSchemaRangeElementType) *RangeElementType {
```

Comment:
nit: use a different name for input parameter `bq`

Reply:
done